sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/src/spanWrapper.ts

```diff
@@ -30,6 +30,10 @@ export class OpenTelemetrySpanWrapper implements TracingSpan {
     }
   }
 
+  addEvent(name: string, attributesOrStartTime?: unknown, startTime?: unknown): void {
```

Comment:
https://github.com/Azure/azure-sdk-for-js/pull/31162/files will add support for this which you can then use (once merged) - we will need to publish a new beta as well